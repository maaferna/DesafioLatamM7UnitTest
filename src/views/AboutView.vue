<template>
    <div class="about">
      <h1>About Us</h1>
      <p>Learn more about our team and mission!</p>
    </div>
  </template>
  
  <script>
  export default {
    name: 'AboutView',
  };
  </script>
  
  <style scoped>
  .about {
    text-align: center;
    padding: 20px;
  }
  </style>
  