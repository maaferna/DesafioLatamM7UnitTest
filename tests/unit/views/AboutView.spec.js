import { mount } from '@vue/test-utils';
import AboutView from '@/views/AboutView.vue';

describe('AboutView', () => {
  test('renders the About view', () => {
    const wrapper = mount(AboutView);
    expect(wrapper.text()).toContain('About Us');
  });

  test('matches snapshot', () => {
    const wrapper = mount(AboutView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
