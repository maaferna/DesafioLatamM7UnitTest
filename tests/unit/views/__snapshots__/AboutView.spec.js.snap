// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView matches snapshot 1`] = `
<div class="about">
  <h1>About Us</h1>
  <p>Learn more about our team and mission!</p>
</div>
`;
